--- a/plugins/generic/staticPages/templates/content.tpl
+++ b/plugins/generic/staticPages/templates/content.tpl
@@ -9,8 +9,10 @@
  *}
 {include file="frontend/components/header.tpl" pageTitleTranslated=$title}
 
+{include file="frontend/components/breadcrumbs.tpl" currentTitle=$title}
+
 <div class="page">
-	<h2>{$title|escape}</h2>
+	<h1 class="page_title">{$title|escape}</h1>
 	{$content}
 </div>
 
