--- a/lib/pkp/plugins/importexport/native/PKPNativeImportExportPlugin.php
+++ b/lib/pkp/plugins/importexport/native/PKPNativeImportExportPlugin.php
@@ -187,6 +187,12 @@
                 $this->isResultManaged = true;
                 break;
             case 'exportSubmissionsBounce':
+                if (empty($request->getUserVar('selectedSubmissions'))) {
+                    $this->result = (new JSONMessage(false, __('plugins.importexport.common.error.noObjectsSelected')))->getString();
+                    $this->isResultManaged = true;
+                    break;
+                }
+
                 $tab = $this->getBounceTab(
                     $request,
                     __('plugins.importexport.native.export.submissions.results'),
--- a/plugins/importexport/onix30/Onix30ExportPlugin.php
+++ b/plugins/importexport/onix30/Onix30ExportPlugin.php
@@ -20,6 +20,7 @@
 use APP\core\Request;
 use APP\template\TemplateManager;
 use BadMethodCallException;
+use PKP\core\JSONMessage;
 use PKP\core\PKPApplication;
 use PKP\plugins\ImportExportPlugin;
 use stdClass;
@@ -129,6 +130,10 @@
                 $templateMgr->display($this->getTemplateResource('index.tpl'));
                 break;
             case 'exportSubmissionsBounce':
+                if (empty($request->getUserVar('selectedSubmissions'))) {
+                    return (new JSONMessage(false, __('plugins.importexport.common.error.noObjectsSelected')))->getString();
+                }
+
                 $tab = $this->getBounceTab(
                     $request,
                     __('plugins.importexport.native.export.submissions.results'),
