--- a/plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.php
+++ b/plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.php
@@ -109,7 +109,7 @@
         $this->addLocalizedElements($dc11Description, 'dc:subject', $subjects);
 
         // Description
-        $this->addLocalizedElements($dc11Description, 'dc:description', $publication->getData('abstract'));
+        $this->addLocalizedElements($dc11Description, 'dc:description', (array) $publication->getData('abstract'));
 
         // Publisher
         $publisher = $press->getData('publisher');
@@ -307,7 +307,7 @@
     private function addLocalizedElements(
         MetadataDescription &$description,
         string $propertyName,
-        array $localizedValues
+        ?array $localizedValues
     ): void {
         foreach (stripAssocArray((array) $localizedValues) as $locale => $values) {
             if (is_scalar($values)) {
