--- a/templates/frontend/objects/monograph_full.tpl
+++ b/templates/frontend/objects/monograph_full.tpl
@@ -580,12 +580,15 @@
 								{assign var=pubIdType value=$pubIdPlugin->getPubIdType()}
 								{assign var=storedPubId value=$publicationFormat->getStoredPubId($pubIdType)}
 								{if $storedPubId != ''}
+									{assign var=resolvingUrl value=$pubIdPlugin->getResolvingURL($currentContext->getId(), $storedPubId)}
 									<div class="sub_item pubid {$publicationFormat->getId()|escape}">
 										<h2 class="label">
-											{$pubIdType}
+											{$pubIdPlugin->getPubIdDisplayType()|escape}
 										</h2>
 										<div class="value">
-											{$storedPubId|escape}
+											<a href="{$resolvingUrl|escape}">
+												{$resolvingUrl|escape}
+											</a>
 										</div>
 									</div>
 								{/if}
