--- a/lib/ui-library/src/pages/jobs/FailedJobsPage.vue
+++ b/lib/ui-library/src/pages/jobs/FailedJobsPage.vue
@@ -123,7 +123,10 @@
 				headers: {
 					'X-Csrf-Token': pkp.currentUser.csrfToken,
 				},
-				error: this.ajaxErrorCallback,
+				error: (r) => {
+					this.isLoadingItems = false;
+					this.ajaxErrorCallback(r);
+				},
 				success: (response) => {
 					pkp.eventBus.$emit('notify', response.message, 'success');
 					this.loadList(1);
--- a/lib/ui-library/src/pages/jobs/JobsPageBase.vue
+++ b/lib/ui-library/src/pages/jobs/JobsPageBase.vue
@@ -66,6 +66,8 @@
 					this.total = response.total;
 					this.currentPage = response.pagination.currentPage;
 					this.lastPage = response.pagination.lastPage;
+				},
+				complete: () => {
 					this.isLoadingItems = false;
 				},
 			});
