--- a/plugins/pubIds/urn/classes/form/URNSettingsForm.php
+++ b/plugins/pubIds/urn/classes/form/URNSettingsForm.php
@@ -73,19 +73,19 @@
             return $form->getData('enableIssueURN') || $form->getData('enablePublicationURN') || $form->getData('enableRepresentationURN');
         }));
         $this->addCheck(new \PKP\form\validation\FormValidatorRegExp($this, 'urnPrefix', 'required', 'plugins.pubIds.urn.manager.settings.form.urnPrefixPattern', '/^urn:[a-zA-Z0-9-]*:.*/'));
-        $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'urnIssueSuffixPattern', 'required', 'plugins.pubIds.urn.manager.settings.form.urnIssueSuffixPatternRequired', function ($urnIssueSuffixPattern) use ($form) {
+        $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'urnIssueSuffixPattern', 'required', 'plugins.pubIds.urn.manager.settings.urnIssueSuffixPatternRequired', function ($urnIssueSuffixPattern) use ($form) {
             if ($form->getData('urnSuffix') == 'pattern' && $form->getData('enableIssueURN')) {
                 return $urnIssueSuffixPattern != '';
             }
             return true;
         }));
-        $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'urnPublicationSuffixPattern', 'required', 'plugins.pubIds.urn.manager.settings.form.urnPublicationSuffixPatternRequired', function ($urnPublicationSuffixPattern) use ($form) {
+        $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'urnPublicationSuffixPattern', 'required', 'plugins.pubIds.urn.manager.settings.urnPublicationSuffixPatternRequired', function ($urnPublicationSuffixPattern) use ($form) {
             if ($form->getData('urnSuffix') == 'pattern' && $form->getData('enablePublicationURN')) {
                 return $urnPublicationSuffixPattern != '';
             }
             return true;
         }));
-        $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'urnRepresentationSuffixPattern', 'required', 'plugins.pubIds.urn.manager.settings.form.urnRepresentationSuffixPatternRequired', function ($urnRepresentationSuffixPattern) use ($form) {
+        $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'urnRepresentationSuffixPattern', 'required', 'plugins.pubIds.urn.manager.settings.urnRepresentationSuffixPatternRequired', function ($urnRepresentationSuffixPattern) use ($form) {
             if ($form->getData('urnSuffix') == 'pattern' && $form->getData('enableRepresentationURN')) {
                 return $urnRepresentationSuffixPattern != '';
             }
