--- a/plugins/blocks/browse/templates/block.tpl
+++ b/plugins/blocks/browse/templates/block.tpl
@@ -15,9 +15,9 @@
  *
  *}
 <div class="pkp_block block_browse">
-	<span class="title">
+	<h2 class="title">
 		{translate key="plugins.block.browse"}
-	</span>
+	</h2>
 
 	<nav class="content" role="navigation" aria-label="{translate|escape key="plugins.block.browse"}">
 		<ul>
--- a/plugins/themes/default/styles/sidebar.less
+++ b/plugins/themes/default/styles/sidebar.less
@@ -17,6 +17,7 @@
 	.title {
 		display: block;
 		margin-bottom: @base;
+		margin-top: 0;
 		font-family: @font-heading;
 		font-size: @font-bump;
 		font-weight: @bold;
