--- a/lib/ui-library/src/managers/UserAccessManager/UserAccessManagerCellActions.vue
+++ b/lib/ui-library/src/managers/UserAccessManager/UserAccessManagerCellActions.vue
@@ -2,7 +2,7 @@
 	<TableCell>
 		<DropdownActions
 			:actions="store.getItemActions({user})"
-			:label="t('userAccess.management.options')"
+			:label="t('common.moreActions')"
 			button-variant="ellipsis"
 			direction="left"
 			@action="(actionName) => store[actionName]({user})"
