--- a/schemas/context.json
+++ b/schemas/context.json
@@ -92,6 +92,12 @@
 				"nullable"
 			]
 		},
+		"restrictPreprintAccess": {
+			"type": "boolean",
+			"validation": [
+				"nullable"
+			]
+		},
 		"doiVersioning": {
 			"type": "boolean",
 			"default": true,
--- a/pages/preprint/PreprintHandler.php
+++ b/pages/preprint/PreprintHandler.php
@@ -34,6 +34,7 @@
 use PKP\plugins\Hook;
 use PKP\plugins\PluginRegistry;
 use PKP\security\authorization\ContextRequiredPolicy;
+use PKP\security\Validation;
 use PKP\submission\Genre;
 use PKP\submission\GenreDAO;
 use PKP\submission\PKPSubmission;
@@ -459,6 +460,10 @@
 
         // If the preprint is posted OR author or server manager who can view unposted preprints
         if (($submission && $submission->getData('status') === PKPSubmission::STATUS_PUBLISHED) || ($submission && $user && Repo::submission()->canPreview($user, $submission))) {
+            // Check if login is required for viewing.
+            if (!$user && $request->getContext()->getData('restrictPreprintAccess')) {
+                Validation::redirectLogin();
+            }
             return true;
         }
         return false;
