--- a/lib/pkp/js/controllers/grid/settings/user/form/UserDetailsFormHandler.js
+++ b/lib/pkp/js/controllers/grid/settings/user/form/UserDetailsFormHandler.js
@@ -89,7 +89,7 @@
 		}
 		$(':password', $form).
 				prop('disabled', activeAndCheck).val(passwordValue);
-		$('[id^="sendNotify"]', $form).attr('disabled', activeAndCheck).
+		$('[id^="sendNotify"]', $form).prop('disabled', activeAndCheck).
 				prop('checked', activeAndCheck);
 	};
 
