--- a/classes/oai/ojs/OAIDAO.php
+++ b/classes/oai/ojs/OAIDAO.php
@@ -25,10 +25,10 @@
 use APP\journal\JournalDAO;
 use APP\publication\enums\VersionStage;
 use APP\section\Section;
-use DateTime;
 use Illuminate\Database\Query\Builder;
 use Illuminate\Support\Facades\DB;
 use PKP\context\Context;
+use PKP\core\Core;
 use PKP\db\DAO;
 use PKP\db\DAORegistry;
 use PKP\galley\DAO as PKPGalleyDAO;
@@ -326,17 +326,17 @@
                 return $query->where('p.section_id', '=', (int) $sectionId);
             })
             ->when($from, function ($query, $from) {
-                return $query->whereDate(
+                return $query->where(
                     DB::raw('GREATEST(a.last_modified, i.last_modified, p.last_modified)'),
                     '>=',
-                    DateTime::createFromFormat('U', $from)
+                    Core::getCurrentDate($from)
                 );
             })
             ->when($until, function ($query, $until) {
-                return $query->whereDate(
+                return $query->where(
                     DB::raw('GREATEST(a.last_modified, i.last_modified, p.last_modified)'),
                     '<=',
-                    DateTime::createFromFormat('U', $until)
+                    Core::getCurrentDate($until)
                 );
             })
             ->when($submissionId, function ($query, $submissionId) {
@@ -391,17 +391,17 @@
                     return $query->where('p.section_id', '=', (int) $sectionId);
                 })
                 ->when($from, function ($query, $from) {
-                    return $query->whereDate(
+                    return $query->where(
                         DB::raw('GREATEST(a.last_modified, i.last_modified, p.last_modified)'),
                         '>=',
-                        DateTime::createFromFormat('U', $from)
+                        Core::getCurrentDate($from)
                     );
                 })
                 ->when($until, function ($query, $until) {
-                    return $query->whereDate(
+                    return $query->where(
                         DB::raw('GREATEST(a.last_modified, i.last_modified, p.last_modified)'),
                         '<=',
-                        DateTime::createFromFormat('U', $until)
+                        Core::getCurrentDate($until)
                     );
                 })
                 ->when($submissionId, function ($query, $submissionId) {
@@ -456,10 +456,10 @@
                     ->orWhere('dot.set_spec', 'like', $set . ':%');
             })
             ->when($from, function ($query, $from) {
-                return $query->whereDate('dot.date_deleted', '>=', DateTime::createFromFormat('U', $from));
+                return $query->where('dot.date_deleted', '>=', Core::getCurrentDate($from));
             })
             ->when($until, function ($query, $until) {
-                return $query->whereDate('dot.date_deleted', '<=', DateTime::createFromFormat('U', $until));
+                return $query->where('dot.date_deleted', '<=', Core::getCurrentDate($until));
             })
             ->when($submissionId, function ($query, $submissionId) {
                 return $query->where('dot.data_object_id', '=', (int) $submissionId);
