--- a/classes/oai/ops/OAIDAO.php
+++ b/classes/oai/ops/OAIDAO.php
@@ -23,6 +23,7 @@
 use APP\server\ServerDAO;
 use Illuminate\Database\Query\Builder;
 use Illuminate\Support\Facades\DB;
+use PKP\core\Core;
 use PKP\db\DAO;
 use PKP\db\DAORegistry;
 use PKP\galley\DAO as PKPGalleyDAO;
@@ -242,10 +243,10 @@
                 return $query->where('p.section_id', '=', $sectionId);
             })
             ->when($from, function ($query, $from) {
-                return $query->whereDate('a.last_modified', '>=', \DateTime::createFromFormat('U', $from));
+                return $query->where('a.last_modified', '>=', Core::getCurrentDate($from));
             })
             ->when($until, function ($query, $until) {
-                return $query->whereDate('a.last-modified', '<=', \DateTime::createFromFormat('U', $until));
+                return $query->where('a.last_modified', '<=', Core::getCurrentDate($until));
             })
             ->when($submissionId, function ($query, $submissionId) {
                 return $query->where('a.submission_id', '=', $submissionId);
@@ -284,10 +285,10 @@
                             ->orWhere('dot.set_spec', 'like', $set . ':%');
                     })
                     ->when($from, function ($query, $from) {
-                        return $query->whereDate('dot.date_deleted', '>=', \DateTime::createFromFormat('U', $from));
+                        return $query->where('dot.date_deleted', '>=', Core::getCurrentDate($from));
                     })
                     ->when($until, function ($query, $until) {
-                        return $query->whereDate('dot.date_deleted', '<=', \DateTime::createFromFormat('U', $until));
+                        return $query->where('dot.date_deleted', '<=', Core::getCurrentDate($until));
                     })
                     ->when($submissionId, function ($query, $submissionId) {
                         return $query->where('dot.data_object_id', '=', (int) $submissionId);
