--- a/lib/pkp/js/controllers/grid/users/stageParticipant/form/StageParticipantNotifyHandler.js
+++ b/lib/pkp/js/controllers/grid/users/stageParticipant/form/StageParticipantNotifyHandler.js
@@ -315,6 +315,7 @@
 		if ($(sourceElement).prop('name') == 'userGroupId') {
 			$checkbox.attr('disabled', 'disabled');
 			$checkbox.removeAttr('checked');
+			$checkbox.prop('checked', false);
 			$checkboxDiv.hide();
 		} else if ($(sourceElement).prop('name') == 'userIdSelected' &&
 				!$checkboxDiv.is(':visible')) {
