--- a/plugins/generic/pdfJsViewer/PdfJsViewerPlugin.php
+++ b/plugins/generic/pdfJsViewer/PdfJsViewerPlugin.php
@@ -146,6 +146,7 @@
                 'galleyTitle' => $galleyTitle,
                 'datePublished' => $datePublished,
                 'isTitleHtml' => true,
+                'isIssueGalley' => false,
             ]);
 
             $templateMgr->display($this->getTemplateResource('display.tpl'));
@@ -209,6 +210,7 @@
                 'datePublished' => $datePublished,
                 'title' => $title,
                 'isTitleHtml' => false,
+                'isIssueGalley' => true,
             ]);
 
             $templateMgr->display($this->getTemplateResource('display.tpl'));
--- a/plugins/generic/pdfJsViewer/templates/display.tpl
+++ b/plugins/generic/pdfJsViewer/templates/display.tpl
@@ -33,7 +33,7 @@
 
 		<a href="{$parentUrl}" class="return">
 			<span class="pkp_screen_reader">
-				{if $issue}
+				{if $isIssueGalley}
 					{translate key="issue.return"}
 				{else}
 					{translate key="article.return"}
