--- a/pages/submission/SubmissionHandler.php
+++ b/pages/submission/SubmissionHandler.php
@@ -176,6 +176,7 @@
             'galleys' => Repo::galley()
                 ->getSchemaMap($submission, $publication, $genres)
                 ->mapMany($galleys)
+                ->values()
         ]);
 
         Hook::add('Template::SubmissionWizard::Section', function (string $hookName, array $params) {
