--- a/plugins/generic/staticPages/cypress/tests/functional/StaticPages.cy.js
+++ b/plugins/generic/staticPages/cypress/tests/functional/StaticPages.cy.js
@@ -46,7 +46,8 @@
 
 		// View the static page
 		cy.visit('/index.php/publicknowledge/flarm');
-		cy.get('h2:contains("Test Static Page")');
+		cy.get('nav.cmp_breadcrumbs li.current:contains("Test Static Page")');
+		cy.get('.page h1:contains("Test Static Page")');
 		cy.get('p:contains("Here is my new static page.")');
 	});
 })
--- a/plugins/generic/staticPages/templates/content.tpl
+++ b/plugins/generic/staticPages/templates/content.tpl
@@ -10,7 +10,8 @@
 {include file="frontend/components/header.tpl" pageTitleTranslated=$title}
 
 <div class="page">
-	<h2>{$title|escape}</h2>
+	{include file="frontend/components/breadcrumbs.tpl" currentTitle=$title}
+	<h1 class="page_title">{$title|escape}</h1>
 	{$content}
 </div>
 
