--- a/plugins/pubIds/urn/URNPubIdPlugin.php
+++ b/plugins/pubIds/urn/URNPubIdPlugin.php
@@ -381,7 +381,7 @@
             return;

             // Use a simplified view when only assigning to the publication
-        } elseif ($publicationFormatUrnEnabled && !$chapterUrnEnabled && !$publicationFormatUrnEnabled && !$submissionFileUrnEnabled) {
+        } elseif ($publicationUrnEnabled && !$chapterUrnEnabled && !$publicationFormatUrnEnabled && !$submissionFileUrnEnabled) {
             if ($form->publication->getData('pub-id::other::urn')) {
                 $msg = __('plugins.pubIds.urn.editor.preview.publication', ['urn' => $form->publication->getData('pub-id::other::urn')]);
             } else {
