--- a/plugins/reports/subscriptions/SubscriptionReportPlugin.php
+++ b/plugins/reports/subscriptions/SubscriptionReportPlugin.php
@@ -20,7 +20,6 @@
 use APP\subscription\SubscriptionTypeDAO;
 use PKP\core\PKPString;
 use PKP\db\DAORegistry;
-use PKP\facades\Locale;
 use PKP\plugins\ReportPlugin;
 
 class SubscriptionReportPlugin extends ReportPlugin
@@ -78,7 +77,6 @@
         $subscriptionTypeDao = DAORegistry::getDAO('SubscriptionTypeDAO'); /** @var SubscriptionTypeDAO $subscriptionTypeDao */
         $individualSubscriptionDao = DAORegistry::getDAO('IndividualSubscriptionDAO'); /** @var IndividualSubscriptionDAO $individualSubscriptionDao */
         $institutionalSubscriptionDao = DAORegistry::getDAO('InstitutionalSubscriptionDAO'); /** @var InstitutionalSubscriptionDAO $institutionalSubscriptionDao */
-        $countries = Locale::getCountries();
 
         header('content-type: text/comma-separated-values');
         header('content-disposition: attachment; filename=subscriptions-' . date('Ymd') . '.csv');
@@ -157,12 +155,7 @@
                         $columns[$index] = PKPString::html2text($user->getMailingAddress());
                         break;
                     case 'country':
-                        $userCountry = $user->getCountry();
-                        $country = null;
-                        if ($userCountry) {
-                            $country = $countries->getByAlpha2($user->getCountry());
-                        }
-                        $columns[$index] = $country ? $country->getLocalName() : '';
+                        $columns[$index] = $user->getCountryLocalized() ?? '';
                         break;
                     case 'email':
                         $columns[$index] = $user->getEmail();
@@ -257,8 +250,7 @@
                         $columns[$index] = PKPString::html2text($user->getMailingAddress());
                         break;
                     case 'country':
-                        $country = $countries->getByAlpha2($user->getCountry());
-                        $columns[$index] = $country ? $country->getLocalName() : '';
+                        $columns[$index] = $user->getCountryLocalized() ?? '';
                         break;
                     case 'email':
                         $columns[$index] = $user->getEmail();
