--- a/templates/frontend/pages/purchaseIndividualSubscription.tpl
+++ b/templates/frontend/pages/purchaseIndividualSubscription.tpl
@@ -14,6 +14,8 @@
 	<form class="cmp_form purchase_subscription" method="post" id="subscriptionForm" action="{url op="payPurchaseSubscription" path="individual"|to_array:$subscriptionId}">
 		{csrf}
 
+		{include file="common/formErrors.tpl"}
+
 		<fieldset>
 			<legend>
 				{translate key="payment.subscription.purchase"}
