--- a/pages/issue/IssueHandler.php
+++ b/pages/issue/IssueHandler.php
@@ -298,7 +298,7 @@
                         $dateEndMembership = $user->getData('dateEndMembership', 0);
                         if ($completedPaymentDao->hasPaidPurchaseIssue($userId, $issue->getId()) || (!is_null($dateEndMembership) && $dateEndMembership > time())) {
                             return true;
-                        } else {
+                        } elseif ($paymentManager->purchaseIssueEnabled()) {
                             // Otherwise queue an issue purchase payment and display payment form
                             $queuedPayment = $paymentManager->createQueuedPayment($request, OJSPaymentManager::PAYMENT_TYPE_PURCHASE_ISSUE, $userId, $issue->getId(), $journal->getData('purchaseIssueFee'));
                             $paymentManager->queuePayment($queuedPayment);
