--- a/classes/oai/omp/PressOAI.php
+++ b/classes/oai/omp/PressOAI.php
@@ -73,11 +73,10 @@
     public function identifierToPublicationFormatId(string $identifier): false|int
     {
         $prefix = $this->getIdentifierPrefix();
-        if (strstr($identifier, $prefix)) {
-            return (int) str_replace($prefix, '', $identifier);
-        } else {
+        if (!str_starts_with($identifier, $prefix) || !preg_match('#^(\d+)\z#', substr($identifier, strlen($prefix)), $matches)) {
             return false;
         }
+        return (int) $matches[1];
     }
 
     /**
