--- a/classes/publication/Repository.php
+++ b/classes/publication/Repository.php
@@ -93,21 +93,21 @@
                     }
                 }
             }
-        }
 
-        if (isset($props['plainLanguageSummary'])) {
-            // Check the word count on plain language summary
-            $wordCountLimit = $section->getData('wordCount');
-            if ($wordCountLimit) {
-                $plainLanguageSummaryErrors = $this->validateWordCount(
-                    $context,
-                    $submission,
-                    $wordCountLimit,
-                    'publication.plainLanguageSummary.wordCountLong',
-                    $props['plainLanguageSummary']
-                );
-                if (count($plainLanguageSummaryErrors)) {
-                    $errors['plainLanguageSummary'] = $plainLanguageSummaryErrors;
+            if (isset($props['plainLanguageSummary'])) {
+                // Check the word count on plain language summary, like the abstract's above
+                $wordCountLimit = $section->getData('wordCount');
+                if ($wordCountLimit) {
+                    $plainLanguageSummaryErrors = $this->validateWordCount(
+                        $context,
+                        $submission,
+                        $wordCountLimit,
+                        'publication.plainLanguageSummary.wordCountLong',
+                        $props['plainLanguageSummary']
+                    );
+                    if (count($plainLanguageSummaryErrors)) {
+                        $errors['plainLanguageSummary'] = $plainLanguageSummaryErrors;
+                    }
                 }
             }
         }
