--- a/plugins/generic/pdfJsViewer/templates/display.tpl
+++ b/plugins/generic/pdfJsViewer/templates/display.tpl
@@ -48,31 +48,6 @@
 
 	</header>
 
-	<script type="text/javascript" src="{$pluginUrl}/pdf.js/build/pdf.js"></script>
-	<script type="text/javascript">
-		{literal}
-			$(document).ready(function() {
-				PDFJS.workerSrc='{/literal}{$pluginUrl}/pdf.js/build/pdf.worker.js{literal}';
-				PDFJS.getDocument({/literal}'{$downloadUrl|escape:"javascript"}'{literal}).then(function(pdf) {
-					// Using promise to fetch the page
-					pdf.getPage(1).then(function(page) {
-						var pdfCanvasContainer = $('#pdfCanvasContainer');
-						var canvas = document.getElementById('pdfCanvas');
-						canvas.height = pdfCanvasContainer.height();
-						canvas.width = pdfCanvasContainer.width()-2; // 1px border each side
-						var viewport = page.getViewport(canvas.width / page.getViewport(1.0).width);
-						var context = canvas.getContext('2d');
-						var renderContext = {
-							canvasContext: context,
-							viewport: viewport
-						};
-						page.render(renderContext);
-					});
-				});
-			});
-		{/literal}
-	</script>
-	<script type="text/javascript" src="{$pluginUrl}/pdf.js/web/viewer.js"></script>
 	<script type="text/javascript">
 		// Creating iframe's src in JS instead of Smarty so that EZProxy-using sites can find our domain in $pdfUrl and do their rewrites on it.
 		$(document).ready(function() {ldelim}
