--- a/lib/pkp/templates/controllers/grid/users/stageParticipant/addParticipantForm.tpl
+++ b/lib/pkp/templates/controllers/grid/users/stageParticipant/addParticipantForm.tpl
@@ -57,6 +57,7 @@
 			{/if}
 
 			{if !$isChangePermitMetadataAllowed && !$isChangeRecommendOnlyAllowed}
+				{assign var="noOptionsToHandle" value=true}
 				{translate key="stageParticipants.noOptionsToHandle"}
 			{/if}
 		{else}
@@ -85,5 +86,5 @@
 			<p><span class="formRequired">{translate key="common.requiredField"}</span></p>
 		{/fbvFormArea}
 	{/if}
-	{fbvFormButtons}
+	{fbvFormButtons submitDisabled=$noOptionsToHandle|default:false}
 </form>
