--- a/templates/frontend/objects/preprint_summary.tpl
+++ b/templates/frontend/objects/preprint_summary.tpl
@@ -29,7 +29,7 @@
 <div class="obj_preprint_summary">
 	{if $preprint->getCurrentPublication()->getLocalizedData('coverImage')}
 		<div class="cover">
-			<a {if $server}href="{url server=$server->getPath() page="preprint" op="view" path=$preprintPath}"{else}href="{url page="preprint" op="view" path=$preprintPath}"{/if} class="file">
+			<a {if $server}href="{url server=$server->getPath() page="preprint" op="view" path=$preprintPath}"{else}href="{url page="preprint" op="view" path=$preprintPath}"{/if} class="file" aria-hidden="true" tabindex="-1">
 				{assign var="coverImage" value=$preprint->getCurrentPublication()->getLocalizedData('coverImage')}
 				<img
 					src="{$preprint->getCurrentPublication()->getLocalizedCoverImageUrl($preprint->getData('contextId'))|escape}"
