--- a/pages/catalog/CatalogBookHandler.php
+++ b/pages/catalog/CatalogBookHandler.php
@@ -530,7 +530,7 @@
                     publicationFormat: $publicationFormat,
                     submissionFile: $submissionFile,
                     chapter: $chapter,
-                    publication: $this->publication,
+                    publication: $publication,
                 ));
             }
             $returner = true;
