--- a/classes/components/forms/context/AccessForm.php
+++ b/classes/components/forms/context/AccessForm.php
@@ -38,24 +38,15 @@
         $this->action = $action;
         $this->locales = $locales;
 
-        $this->addField(new FieldOptions('publishingMode', [
-            'label' => __('manager.distribution.publishingMode'),
+        $this->addField(new FieldOptions('enableOai', [
+            'label' => __('manager.setup.enableOai'),
+            'description' => __('manager.setup.enableOai.description'),
             'type' => 'radio',
             'options' => [
-                ['value' => Server::PUBLISHING_MODE_OPEN, 'label' => __('manager.distribution.publishingMode.openAccess')],
-                ['value' => Server::PUBLISHING_MODE_NONE, 'label' => __('manager.distribution.publishingMode.none')],
+                ['value' => true, 'label' => __('common.enable')],
+                ['value' => false, 'label' => __('common.disable')],
             ],
-            'value' => $context->getData('publishingMode'),
-        ]))
-            ->addField(new FieldOptions('enableOai', [
-                'label' => __('manager.setup.enableOai'),
-                'description' => __('manager.setup.enableOai.description'),
-                'type' => 'radio',
-                'options' => [
-                    ['value' => true, 'label' => __('common.enable')],
-                    ['value' => false, 'label' => __('common.disable')],
-                ],
-                'value' => $context->getData('enableOai'),
-            ]));
+            'value' => $context->getData('enableOai'),
+        ]));
     }
 }
