--- a/lib/pkp/classes/submissionFile/Repository.php	2026-09-30 21:03:04.364152942 +0000
+++ b/lib/pkp/classes/submissionFile/Repository.php	2026-09-30 21:03:04.392153236 +0000
@@ -243,6 +243,22 @@
             });
         }
 
+        // Make sure that the genre is one of the submission's context
+        if (isset($props['genreId'])) {
+            $validator->after(function ($validator) use ($object, $props) {
+                if ($validator->errors()->get('genreId')) {
+                    return;
+                }
+                $submissionId = $props['submissionId'] ?? $object?->getData('submissionId');
+                $submission = $submissionId ? Repo::submission()->get((int) $submissionId) : null;
+                /** @var \PKP\submission\GenreDAO $genreDao */
+                $genreDao = DAORegistry::getDAO('GenreDAO');
+                if (!$submission || !$genreDao->getById($props['genreId'], $submission->getData('contextId'))) {
+                    $validator->errors()->add('genreId', __('submission.upload.noGenre'));
+                }
+            });
+        }
+
         $errors = [];
 
         if ($validator->fails()) {
