--- a/lib/pkp/api/v1/submissions/tasks/EditorialTaskController.php
+++ b/lib/pkp/api/v1/submissions/tasks/EditorialTaskController.php
@@ -1100,7 +1100,7 @@
             ));
 
             Mail::send($mailable);
-            Repo::emailLogEntry()->logMailable(SubmissionEmailLogEventType::DISCUSSION_NOTIFY, $mailable, $submission);
+            Repo::emailLogEntry()->logMailable(SubmissionEmailLogEventType::DISCUSSION_NOTIFY, $mailable, $submission, $currentUser);
         }
     }
 
--- a/lib/pkp/controllers/grid/users/stageParticipant/form/PKPStageParticipantNotifyForm.php
+++ b/lib/pkp/controllers/grid/users/stageParticipant/form/PKPStageParticipantNotifyForm.php
@@ -268,28 +268,28 @@
         switch ($templateKey) {
             case 'EDITOR_ASSIGN':
                 $this->_addAssignmentTaskNotification($request, Notification::NOTIFICATION_TYPE_EDITOR_ASSIGN, $user->getId(), $submission->getId());
-                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::EDITOR_ASSIGN, $mailable, $submission);
+                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::EDITOR_ASSIGN, $mailable, $submission, $request->getUser());
                 break;
             case 'COPYEDIT_REQUEST':
                 $this->_addAssignmentTaskNotification($request, Notification::NOTIFICATION_TYPE_COPYEDIT_ASSIGNMENT, $user->getId(), $submission->getId());
-                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::COPYEDIT_NOTIFY_COPYEDITOR, $mailable, $submission);
+                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::COPYEDIT_NOTIFY_COPYEDITOR, $mailable, $submission, $request->getUser());
                 break;
             case 'LAYOUT_REQUEST':
                 $this->_addAssignmentTaskNotification($request, Notification::NOTIFICATION_TYPE_LAYOUT_ASSIGNMENT, $user->getId(), $submission->getId());
-                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::LAYOUT_NOTIFY_EDITOR, $mailable, $submission);
+                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::LAYOUT_NOTIFY_EDITOR, $mailable, $submission, $request->getUser());
                 break;
             case 'INDEX_REQUEST':
                 $this->_addAssignmentTaskNotification($request, Notification::NOTIFICATION_TYPE_INDEX_ASSIGNMENT, $user->getId(), $submission->getId());
-                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::INDEX_NOTIFY_INDEXER, $mailable, $submission);
+                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::INDEX_NOTIFY_INDEXER, $mailable, $submission, $request->getUser());
                 break;
             case 'LAYOUT_COMPLETE':
-                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::LAYOUT_NOTIFY_COMPLETE, $mailable, $submission);
+                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::LAYOUT_NOTIFY_COMPLETE, $mailable, $submission, $request->getUser());
                 break;
             case 'INDEX_COMPLETE':
-                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::INDEX_NOTIFY_COMPLETE, $mailable, $submission);
+                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::INDEX_NOTIFY_COMPLETE, $mailable, $submission, $request->getUser());
                 break;
             default:
-                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::DISCUSSION_NOTIFY, $mailable, $submission);
+                !$logRepository ?: $logRepository->logMailable(SubmissionEmailLogEventType::DISCUSSION_NOTIFY, $mailable, $submission, $request->getUser());
                 break;
         }
 
