--- a/lib/pkp/classes/user/Repository.php
+++ b/lib/pkp/classes/user/Repository.php
@@ -22,6 +22,7 @@
 use Illuminate\Support\LazyCollection;
 use PKP\context\Context;
 use PKP\context\SubEditorsDAO;
+use PKP\core\Core;
 use PKP\core\PKPApplication;
 use PKP\db\DAORegistry;
 use PKP\facades\Locale;
@@ -107,6 +108,9 @@
     /** @copydoc DAO::insert() */
     public function add(User $user): int
     {
+        if (!$user->getDateRegistered()) {
+            $user->setDateRegistered(Core::getCurrentDate());
+        }
         $id = $this->dao->insert($user);
         Hook::call('User::add', [$user]);
 
