--- a/lib/pkp/classes/context/SubEditorsDAO.php
+++ b/lib/pkp/classes/context/SubEditorsDAO.php
@@ -249,9 +249,13 @@
 
         // Send an email to assigned editors
         // Replaces StageAssignmentDAO::getBySubmissionAndRoleIds
+        $stages = Application::getApplicationStages();
         $editorAssignments = StageAssignment::withSubmissionIds([$submission->getId()])
             ->withRoleIds([Role::ROLE_ID_MANAGER, Role::ROLE_ID_SUB_EDITOR])
-            ->withStageIds([WORKFLOW_STAGE_ID_SUBMISSION])
+            ->withStageIds([
+                // WORKFLOW_STAGE_ID_SUBMISSION for OJS/OMP and WORKFLOW_STAGE_ID_PRODUCTION for OPS, see pkp/pkp-lib#10874
+                array_shift($stages)
+            ])
             ->get();
 
         $emailTemplate = Repo::emailTemplate()->getByKey($context->getId(), EditorAssigned::getEmailTemplateKey());
