--- a/templates/frontend/pages/book.tpl
+++ b/templates/frontend/pages/book.tpl
@@ -17,7 +17,7 @@
 {if $isChapterRequest}
 	{assign var=pageTitle value=$chapter->getLocalizedFullTitle()}
 {else}
-	{assign var=pageTitle value=$publishedSubmission->getCurrentPublication()->getLocalizedFullTitle()}
+	{assign var=pageTitle value=$publication->getLocalizedFullTitle()}
 {/if}
 
 {include file="frontend/components/header.tpl" pageTitleTranslated=$pageTitle}
