--- a/lib/pkp/jobs/notifications/StatisticsReportMail.php
+++ b/lib/pkp/jobs/notifications/StatisticsReportMail.php
@@ -126,7 +126,7 @@
             foreach (Application::getApplicationStages() as $stageId) {
                 $file->fputcsv([
                     __(Application::getWorkflowStageName($stageId), [], $locale),
-                    app()->get('editorialStats')->countActiveByStages($stageId)
+                    app()->get('editorialStats')->countActiveByStages($stageId, ['contextIds' => [$this->contextId]])
                 ]);
             }
 
