--- a/lib/pkp/xml/oai2.xsl
+++ b/lib/pkp/xml/oai2.xsl
@@ -152,7 +152,7 @@
 <xsl:call-template name='xmlstyle' />
 </xsl:template>
 
-<xsl:variable name='identifier' select="substring-before(concat(substring-after(/oai:OAI-PMH/oai:request,'identifier='),'&amp;'),'&amp;')" />
+<xsl:variable name='identifier' select="/oai:OAI-PMH/oai:request/@identifier" />
 
 <xsl:template match="/">
 <html lang="en">
