--- a/templates/submission/review-chapters.tpl
+++ b/templates/submission/review-chapters.tpl
@@ -15,7 +15,7 @@
         <pkp-button
             aria-describedby="review-chapters"
             class="submissionWizard__reviewPanel__edit"
-            @click="openStep('{$step.id}')"
+            @click="openStep('details')"
         >
             {translate key="common.edit"}
         </pkp-button>
