--- a/templates/frontend/objects/preprint_details.tpl
+++ b/templates/frontend/objects/preprint_details.tpl
@@ -207,7 +207,7 @@
 			{if !empty($publication->getLocalizedData('keywords'))}
 			<section class="item keywords">
 				<h2 class="label">
-					{capture assign=translatedKeywords}{translate key="preprint.subject"}{/capture}
+					{capture assign=translatedKeywords}{translate key="common.keywords"}{/capture}
 					{translate key="semicolon" label=$translatedKeywords}
 				</h2>
 				<span class="value">
