--- a/controllers/grid/settings/series/form/SeriesForm.php
+++ b/controllers/grid/settings/series/form/SeriesForm.php
@@ -53,7 +53,7 @@
 
         // Validation checks for this form
         $form = $this;
-        $this->addCheck(new \PKP\form\validation\FormValidatorLocale($this, 'title', 'required', 'manager.setup.form.series.nameRequired'));
+        $this->addCheck(new \PKP\form\validation\FormValidatorLocale($this, 'title', 'required', 'manager.series.form.titleRequired'));
         $this->addCheck(new \PKP\form\validation\FormValidatorISSN($this, 'onlineIssn', 'optional', 'catalog.manage.series.issn.validation'));
         $this->addCheck(new \PKP\form\validation\FormValidatorISSN($this, 'printIssn', 'optional', 'catalog.manage.series.issn.validation'));
         $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'printIssn', 'optional', 'catalog.manage.series.issn.equalValidation', function ($printIssn) use ($form) {
--- a/lib/pkp/controllers/grid/settings/genre/form/GenreForm.php
+++ b/lib/pkp/controllers/grid/settings/genre/form/GenreForm.php
@@ -65,7 +65,7 @@
 
         // Validation checks for this form
         $form = $this;
-        $this->addCheck(new \PKP\form\validation\FormValidatorLocale($this, 'name', 'required', 'manager.setup.form.genre.nameRequired'));
+        $this->addCheck(new \PKP\form\validation\FormValidatorLocale($this, 'name', 'required', 'manager.setup.genres.nameRequired'));
         $this->addCheck(new \PKP\form\validation\FormValidatorCustom($this, 'key', 'optional', 'manager.setup.genres.key.exists', function ($key) use ($context, $form) {
             $genreDao = DAORegistry::getDAO('GenreDAO'); /** @var GenreDAO $genreDao */
             return $key == '' || !$genreDao->keyExists($key, $context->getId(), $form->getGenreId());
--- a/lib/pkp/locale/en/manager.po
+++ b/lib/pkp/locale/en/manager.po
@@ -2005,6 +2005,9 @@
 msgid "manager.setup.genres.key.exists"
 msgstr "The key already exists."
 
+msgid "manager.setup.genres.nameRequired"
+msgstr "A name is required for the component."
+
 msgid "manager.setup.genres.key.alphaNumeric"
 msgstr ""
 "The key can contain only alphanumeric characters, underscores, and hyphens, "
