--- a/controllers/grid/articleGalleys/form/ArticleGalleyForm.php
+++ b/controllers/grid/articleGalleys/form/ArticleGalleyForm.php
@@ -58,6 +58,7 @@
 
         $this->addCheck(new \PKP\form\validation\FormValidator($this, 'label', 'required', 'editor.issues.galleyLabelRequired'));
         $this->addCheck(new \PKP\form\validation\FormValidatorRegExp($this, 'urlPath', 'optional', 'validator.alpha_dash_period', '/^[a-zA-Z0-9]+([\\.\\-_][a-zA-Z0-9]+)*$/'));
+        $this->addCheck(new \PKP\form\validation\FormValidatorUrl($this, 'urlRemote', 'optional', 'validator.url'));
         $this->addCheck(new \PKP\form\validation\FormValidatorPost($this));
         $this->addCheck(new \PKP\form\validation\FormValidatorCSRF($this));
 
