--- a/controllers/grid/catalogEntry/form/PublicationFormatForm.php
+++ b/controllers/grid/catalogEntry/form/PublicationFormatForm.php
@@ -59,6 +59,7 @@
         $this->addCheck(new \PKP\form\validation\FormValidator($this, 'name', 'required', 'grid.catalogEntry.nameRequired'));
         $this->addCheck(new \PKP\form\validation\FormValidator($this, 'entryKey', 'required', 'grid.catalogEntry.publicationFormatRequired'));
         $this->addCheck(new \PKP\form\validation\FormValidatorRegExp($this, 'urlPath', 'optional', 'validator.alpha_dash_period', '/^[a-zA-Z0-9]+([\\.\\-_][a-zA-Z0-9]+)*$/'));
+        $this->addCheck(new \PKP\form\validation\FormValidatorUrl($this, 'remoteURL', 'optional', 'validator.url'));
         $this->addCheck(new \PKP\form\validation\FormValidatorPost($this));
         $this->addCheck(new \PKP\form\validation\FormValidatorCSRF($this));
     }
