--- a/controllers/grid/submissions/ExportPublishedSubmissionsListGridCellProvider.php
+++ b/controllers/grid/submissions/ExportPublishedSubmissionsListGridCellProvider.php
@@ -95,7 +95,7 @@
                             'edit',
                             new AjaxModal(
                                 $dispatcher->url($request, PKPApplication::ROUTE_COMPONENT, null, 'grid.issues.BackIssueGridHandler', 'editIssue', null, ['issueId' => $issue->getId()]),
-                                __('plugins.importexport.common.settings.DOIPluginSettings'),
+                                __('editor.issues.editIssue', ['issueIdentification' => $issue->getIssueIdentification()]),
                             ),
                             $issue->getIssueIdentification(),
                             null
--- a/controllers/grid/pubIds/PubIdExportIssuesListGridCellProvider.php
+++ b/controllers/grid/pubIds/PubIdExportIssuesListGridCellProvider.php
@@ -72,7 +72,7 @@
                         'edit',
                         new AjaxModal(
                             $dispatcher->url($request, PKPApplication::ROUTE_COMPONENT, null, 'grid.issues.BackIssueGridHandler', 'editIssue', null, ['issueId' => $publishedIssue->getId()]),
-                            __('plugins.importexport.common.settings.DOIPluginSettings')
+                            __('editor.issues.editIssue', ['issueIdentification' => $publishedIssue->getIssueIdentification()])
                         ),
                         $publishedIssue->getIssueIdentification(),
                         null
--- a/controllers/grid/pubIds/PubIdExportRepresentationsListGridCellProvider.php
+++ b/controllers/grid/pubIds/PubIdExportRepresentationsListGridCellProvider.php
@@ -98,7 +98,7 @@
                         'edit',
                         new AjaxModal(
                             $dispatcher->url($request, PKPApplication::ROUTE_COMPONENT, null, 'grid.issues.BackIssueGridHandler', 'editIssue', null, ['issueId' => $issue->getId()]),
-                            __('plugins.importexport.common.settings.DOIPluginSettings')
+                            __('editor.issues.editIssue', ['issueIdentification' => $issue->getIssueIdentification()])
                         ),
                         $issue->getIssueIdentification(),
                         null
