--- a/pages/index/IndexHandler.php
+++ b/pages/index/IndexHandler.php
@@ -66,7 +66,7 @@
             'highlights' => $this->getHighlights($press),
         ]);
 
-        $this->_setupAnnouncements($journal ?? $request->getSite(), $templateMgr);
+        $this->_setupAnnouncements($press ?? $request->getSite(), $templateMgr);
         if ($press) {
             // Display the current press home.
             $this->_displayPressIndexPage($press, $request);
@@ -131,8 +131,6 @@
             'displayCreativeCommons' => $press->getSetting('includeCreativeCommons'),
         ]);
 
-        $this->_setupAnnouncements($press, $templateMgr);
-
         // Display Featured Books
         if ($press->getSetting('displayFeaturedBooks')) {
             $featureDao = DAORegistry::getDAO('FeatureDAO'); /** @var FeatureDAO $featureDao */
