--- a/plugins/generic/googleScholar/GoogleScholarPlugin.php
+++ b/plugins/generic/googleScholar/GoogleScholarPlugin.php
@@ -162,10 +162,10 @@
                                 $templateMgr->addHeader('googleScholarIsbn' . $i++, '<meta name="citation_isbn" content="' . htmlspecialchars($identificationCode->getValue()) . '"/>');
                             }
                         }
-                        $this->_setFileUrl($availableFile, $templateMgr, $i, $request, $submission);
+                        $this->_setFileUrl($availableFile, $templateMgr, $request, $submission);
                     } elseif ($isChapterRequest) {
                         if ($chapter->getId() == $availableFile->getData('chapterId')) {
-                            $this->_setFileUrl($availableFile, $templateMgr, $i, $request, $submission);
+                            $this->_setFileUrl($availableFile, $templateMgr, $request, $submission);
                         }
                     }
                 }
@@ -210,14 +210,14 @@
         return __('plugins.generic.googleScholar.description');
     }
 
-    private function _setFileUrl($availableFile, TemplateManager $templateMgr, int $i, \APP\Core\Request $request, \APP\submission\Submission $submission): void
+    private function _setFileUrl($availableFile, TemplateManager $templateMgr, \APP\Core\Request $request, \APP\submission\Submission $submission): void
     {
         switch ($availableFile->getData('mimetype')) {
             case 'application/pdf':
-                $templateMgr->addHeader('googleScholarPdfUrl' . $i++, '<meta name="citation_pdf_url" content="' . $request->getDispatcher()->url($request, Application::ROUTE_PAGE, null, 'catalog', 'download', [$submission->getId(), $availableFile->getData('assocId'), $availableFile->getId()], urlLocaleForPage: '') . '"/>');
+                $templateMgr->addHeader('googleScholarPdfUrl' . $availableFile->getId(), '<meta name="citation_pdf_url" content="' . $request->getDispatcher()->url($request, Application::ROUTE_PAGE, null, 'catalog', 'download', [$submission->getId(), $availableFile->getData('assocId'), $availableFile->getId()], urlLocaleForPage: '') . '"/>');
                 break;
             case 'text/xml' or 'text/html':
-                $templateMgr->addHeader('googleScholarHtmlUrl' . $i++, '<meta name="citation_fulltext_html_url" content="' . $request->getDispatcher()->url($request, Application::ROUTE_PAGE, null, 'catalog', 'download', [$submission->getId(), $availableFile->getData('assocId'), $availableFile->getId()], urlLocaleForPage: '') . '"/>');
+                $templateMgr->addHeader('googleScholarHtmlUrl' . $availableFile->getId(), '<meta name="citation_fulltext_html_url" content="' . $request->getDispatcher()->url($request, Application::ROUTE_PAGE, null, 'catalog', 'download', [$submission->getId(), $availableFile->getData('assocId'), $availableFile->getId()], urlLocaleForPage: '') . '"/>');
                 break;
         }
     }
