--- a/classes/services/ContextService.php
+++ b/classes/services/ContextService.php
@@ -44,6 +44,7 @@
 
         Hook::add('Context::add', [$this, 'afterAddContext']);
         Hook::add('Context::edit', [$this, 'afterEditContext']);
+        Hook::add('Context::delete::before', [$this, 'beforeDeleteContext']);
         Hook::add('Context::delete', [$this, 'afterDeleteContext']);
         Hook::add('Context::validate', [$this, 'validateContext']);
     }
