--- a/plugins/generic/announcementFeed/templates/atom.tpl
+++ b/plugins/generic/announcementFeed/templates/atom.tpl
@@ -13,7 +13,7 @@
 	{* required elements *}
 	<id>{$selfUrl|escape}</id>
 	<title>{$journal->getLocalizedName()|escape:"html"|strip}: {translate key="announcement.announcements"}</title>
-	<updated>{$dateUpdated|date_format:"%Y-%m-%dT%T%z"|regex_replace:"/00$/":":00"}</updated>
+	<updated>{$dateUpdated|date_format:"Y-m-d\TH:i:sP"}</updated>
 
 	{* recommended elements *}
 	{* <author/> *}
@@ -38,7 +38,7 @@
 		{* required elements *}
 		<id>{url page="announcement" op="view" path=$announcement->id}</id>
 		<title>{$announcement->getLocalizedData('fullTitle')|strip|escape:"html"}</title>
-		<updated>{$announcement->datePosted->format("%Y-%m-%dT%T%z")|regex_replace:"/00$/":":00"}</updated>
+		<updated>{$announcement->datePosted->format("Y-m-d\TH:i:sP")}</updated>
 	  	<author>
 			<name>{$journal->getLocalizedName()|strip|escape:"html"}</name>
         </author>
@@ -50,7 +50,7 @@
 		{* optional elements *}
 		{* <category/> *}
 		{* <contributor/> *}
-		<published>{$announcement->datePosted->format("%Y-%m-%dT%T%z")|regex_replace:"/00$/":":00"}</published>
+		<published>{$announcement->datePosted->format("Y-m-d\TH:i:sP")}</published>
 		{* <source/> *}
 		{* <rights/> *}
 	</entry>
--- a/plugins/generic/announcementFeed/templates/rss.tpl
+++ b/plugins/generic/announcementFeed/templates/rss.tpl
@@ -49,7 +49,7 @@
 		<description>{$announcement->getLocalizedData('description')|strip|escape:"html"}</description>
 		{/if}
 		<dc:creator>{$journal->getLocalizedName()|strip|escape:"html"}</dc:creator>
-		<dc:date>{$announcement->datePosted->format("%Y-%m-%d")}</dc:date>
+		<dc:date>{$announcement->datePosted->format("Y-m-d")}</dc:date>
 	</item>
 {/foreach}
 
--- a/plugins/oaiMetadataFormats/marc/templates/record.tpl
+++ b/plugins/oaiMetadataFormats/marc/templates/record.tpl
@@ -11,7 +11,7 @@
 	xmlns="http://www.openarchives.org/OAI/1.1/oai_marc" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://www.openarchives.org/OAI/1.1/oai_marc http://www.openarchives.org/OAI/1.1/oai_marc.xsd">
 	{if $publication->getData('datePublished')}
-		<fixfield id="008">"{$publication->getData('datePublished')|strtotime|date_format:"%y%m%d %Y"}                        eng  "</fixfield>
+		<fixfield id="008">"{$publication->getData('datePublished')|date_format:"ymd Y"}                        eng  "</fixfield>
 	{/if}
 	{if $journal->getData('onlineIssn')}
 		<varfield id="022" i1="#" i2="#">
--- a/plugins/oaiMetadataFormats/marcxml/templates/record.tpl
+++ b/plugins/oaiMetadataFormats/marcxml/templates/record.tpl
@@ -13,7 +13,7 @@
 	xsi:schemaLocation="http://www.loc.gov/MARC21/slim https://www.loc.gov/standards/marcxml/schema/MARC21slim.xsd">
 	<leader>     nmb a2200000Iu 4500</leader>
 	{if $publication->getData('datePublished')}
-		<controlfield tag="008">"{$publication->getData('datePublished')|strtotime|date_format:"%y%m%d %Y"}                        eng  "</controlfield>
+		<controlfield tag="008">"{$publication->getData('datePublished')|date_format:"ymd Y"}                        eng  "</controlfield>
 	{/if}
 	{if $journal->getData('onlineIssn')}
 		<datafield tag="022" ind1="#" ind2="#">
