--- a/plugins/generic/recommendByAuthor/templates/articleFooter.tpl
+++ b/plugins/generic/recommendByAuthor/templates/articleFooter.tpl
@@ -12,20 +12,21 @@
 		<h2>{translate key="plugins.generic.recommendByAuthor.heading"}</h2>
 		<ul>
 			{foreach from=$articlesBySameAuthor item=articleBySameAuthor}
-				{assign var=submission value=$articleBySameAuthor.publishedSubmission}
+				{assign var=submission value=$articleBySameAuthor.submission}
+				{assign var=publication value=$articleBySameAuthor.currentPublication}
 				{assign var=issue value=$articleBySameAuthor.issue}
-				{assign var=journal value=$articleBySameAuthor.journal}
-				{assign var=publication value=$submission->getCurrentPublication()}
+				{assign var=journal value=$articleBySameAuthor.context}
 				<li>
 					{foreach from=$publication->getData('authors') item=author}
 						{$author->getFullName()|escape},
 					{/foreach}
 					<a href="{url router=PKP\core\PKPApplication::ROUTE_PAGE journal=$journal->getPath() page="article" op="view" path=$submission->getBestId() urlLocaleForPage=""}">
 						{$publication->getLocalizedFullTitle(null, 'html')|strip_unsafe_html}
-					</a>,
+					</a>{if $issue},
 					<a href="{url router=PKP\core\PKPApplication::ROUTE_PAGE journal=$journal->getPath() page="issue" op="view" path=$issue->getBestIssueId() urlLocaleForPage=""}">
 						{$journal->getLocalizedName()|escape}: {$issue->getIssueIdentification()|escape}
 					</a>
+					{/if}
 				</li>
 			{/foreach}
 		</ul>
