--- a/pages/preprint/PreprintHandler.php
+++ b/pages/preprint/PreprintHandler.php
@@ -107,9 +107,7 @@
         // publication, redirect to the current URL
         $currentUrlPath = $submission->getBestId();
         if ($currentUrlPath && $currentUrlPath != $urlPath) {
-            $newArgs = $args;
-            $newArgs[0] = $currentUrlPath;
-            $request->redirect(null, $request->getRequestedPage(), $request->getRequestedOp(), $newArgs);
+            $request->redirect(null, $request->getRequestedPage(), $request->getRequestedOp(), [$currentUrlPath, ...$args]);
         }
 
         $this->preprint = $submission;
@@ -345,7 +343,7 @@
                     ];
                 } else {
                     $redirectArgs = [
-                        $preprint->getId(),
+                        $preprint->getBestId(),
                         $this->galley->getBestGalleyId()
                     ];
                 }
