--- a/pages/article/ArticleHandler.php
+++ b/pages/article/ArticleHandler.php
@@ -571,17 +571,15 @@
                 }
 
                 if (!(!$subscriptionRequired || $submission->getCurrentPublication()->getData('accessStatus') == Submission::ARTICLE_ACCESS_OPEN || $subscribedUser || $purchasedIssue)) {
+                    /* if only pdf files are being restricted, then approve all non-pdf galleys
+                     * and continue checking if it is a pdf galley */
+                    if ($paymentManager->onlyPdfEnabled() && $this->galley && !$this->galley->isPdfGalley()) {
+                        $this->issue = $issue;
+                        $this->article = $submission;
+                        return true;
+                    }
+
                     if ($paymentManager->purchaseArticleEnabled() || $paymentManager->membershipEnabled()) {
-                        /* if only pdf files are being restricted, then approve all non-pdf galleys
-                         * and continue checking if it is a pdf galley */
-                        if ($paymentManager->onlyPdfEnabled()) {
-                            if ($this->galley && !$this->galley->isPdfGalley()) {
-                                $this->issue = $issue;
-                                $this->article = $submission;
-                                return true;
-                            }
-                        }
-
                         if (!Validation::isLoggedIn()) {
                             Validation::redirectLogin('payment.loginRequired.forArticle');
                         }
--- a/pages/issue/IssueHandler.php
+++ b/pages/issue/IssueHandler.php
@@ -280,13 +280,13 @@
                     // Check if payments are enabled,
                     $paymentManager = Application::get()->getPaymentManager($journal);
 
-                    if ($paymentManager->purchaseIssueEnabled() || $paymentManager->membershipEnabled()) {
-                        // If only pdf files are being restricted, then approve all non-pdf galleys
-                        // and continue checking if it is a pdf galley
-                        if ($paymentManager->onlyPdfEnabled() && !$galley->isPdfGalley()) {
-                            return true;
-                        }
+                    // If only pdf files are being restricted, then approve all non-pdf galleys
+                    // and continue checking if it is a pdf galley
+                    if ($paymentManager->onlyPdfEnabled() && $galley && !$galley->isPdfGalley()) {
+                        return true;
+                    }
 
+                    if ($paymentManager->purchaseIssueEnabled() || $paymentManager->membershipEnabled()) {
                         if (!Validation::isLoggedIn()) {
                             Validation::redirectLogin('payment.loginRequired.forIssue');
                         }
