--- a/api/v1/reviewers/recommendations/ReviewerRecommendationController.php
+++ b/api/v1/reviewers/recommendations/ReviewerRecommendationController.php
@@ -27,6 +27,7 @@
 use Illuminate\Support\Facades\Route;
 use PKP\core\PKPBaseController;
 use PKP\core\PKPRequest;
+use PKP\security\authorization\CanAccessSettingsPolicy;
 use PKP\security\authorization\ContextAccessPolicy;
 use PKP\security\authorization\UserRolesRequiredPolicy;
 use PKP\security\Role;
@@ -53,7 +54,6 @@
             self::roleAuthorizer([
                 Role::ROLE_ID_SITE_ADMIN,
                 Role::ROLE_ID_MANAGER,
-                Role::ROLE_ID_SUB_EDITOR,
             ]),
         ];
     }
@@ -68,6 +68,7 @@
 
         $this->addPolicy(new UserRolesRequiredPolicy($request), true);
         $this->addPolicy(new ContextAccessPolicy($request, $roleAssignments));
+        $this->addPolicy(new CanAccessSettingsPolicy());
 
         if (in_array($actionName, ['get', 'edit', 'updateStatus', 'delete'])) {
             $this->addPolicy(
