--- a/controllers/grid/issues/form/IssueGalleyForm.php
+++ b/controllers/grid/issues/form/IssueGalleyForm.php
@@ -115,7 +115,7 @@
             if (ctype_digit((string) $publicGalleyId)) {
                 $this->addError('publicGalleyId', __('editor.publicIdentificationNumericNotAllowed', ['publicIdentifier' => $publicGalleyId]));
                 $this->addErrorField('publicGalleyId');
-            } elseif ($journalDao->anyPubIdExists($journal->getId(), 'publisher-id', $publicGalleyId, Application::ASSOC_TYPE_ISSUE_GALLEY, $this->_issueGalley ? $this->_issueGalley->getId() : null, true)) {
+            } elseif ($journalDao->anyPubIdExists($journal->getId(), 'publisher-id', $publicGalleyId, Application::ASSOC_TYPE_ISSUE_GALLEY, $this->_issueGalley?->getId() ?? 0, true)) {
                 $this->addError('publicGalleyId', __('editor.publicIdentificationExistsForTheSameType', ['publicIdentifier' => $publicGalleyId]));
                 $this->addErrorField('publicGalleyId');
             }
