--- a/controllers/api/file/ManageFileApiHandler.php	2026-09-30 21:31:33.342691391 +0000
+++ b/controllers/api/file/ManageFileApiHandler.php	2026-09-30 21:31:33.348017233 +0000
@@ -21,7 +21,6 @@
 use APP\core\Request;
 use APP\template\TemplateManager;
 use PKP\controllers\api\file\PKPManageFileApiHandler;
-use PKP\controllers\tab\pubIds\form\PKPPublicIdentifiersForm;
 use PKP\core\JSONMessage;
 use PKP\db\DAO;
 use PKP\notification\Notification;
@@ -94,7 +93,7 @@
     {
         $submissionFile = $this->getAuthorizedContextObject(Application::ASSOC_TYPE_SUBMISSION_FILE);
         $stageId = $request->getUserVar('stageId');
-        $form = new PKPPublicIdentifiersForm($submissionFile, $stageId);
+        $form = new PublicIdentifiersForm($submissionFile, $stageId);
         $form->readInputData();
         if ($form->validate()) {
             $form->execute();
@@ -116,7 +115,7 @@
     {
         $submissionFile = $this->getAuthorizedContextObject(Application::ASSOC_TYPE_SUBMISSION_FILE);
         $stageId = $request->getUserVar('stageId');
-        $form = new PKPPublicIdentifiersForm($submissionFile, $stageId);
+        $form = new PublicIdentifiersForm($submissionFile, $stageId);
         $form->clearPubId($request->getUserVar('pubIdPlugIn'));
         return new JSONMessage(true);
     }
