--- a/lib/ui-library/src/pages/workflow/components/publication/WorkflowPublicationJats.vue
+++ b/lib/ui-library/src/pages/workflow/components/publication/WorkflowPublicationJats.vue
@@ -56,7 +56,7 @@
 								v-if="workingJatsProps['loadingContentError'] == null"
 								:label="t('publication.jats.makePublic')"
 								:checked="jatsPublicVisibility"
-								:disabled="isUpdatingVisibility"
+								:disabled="isUpdatingVisibility || !canEdit"
 								class="text-sm !inline-flex shrink-0 whitespace-nowrap"
 								@change="handleVisibilityChange($event)"
 							/>
