--- a/classes/oai/ops/OAIDAO.php
+++ b/classes/oai/ops/OAIDAO.php
@@ -245,7 +245,7 @@
                 return $query->whereDate('a.last_modified', '>=', \DateTime::createFromFormat('U', $from));
             })
             ->when($until, function ($query, $until) {
-                return $query->whereDate('a.last-modified', '<=', \DateTime::createFromFormat('U', $until));
+                return $query->whereDate('a.last_modified', '<=', \DateTime::createFromFormat('U', $until));
             })
             ->when($submissionId, function ($query, $submissionId) {
                 return $query->where('a.submission_id', '=', $submissionId);
