--- a/plugins/blocks/subscription/templates/block.tpl
+++ b/plugins/blocks/subscription/templates/block.tpl
@@ -29,10 +29,12 @@
 			{if $individualSubscription->getMembership()}
 				<p class="subscription_membership">({$individualSubscription->getMembership()|escape})</p>
 			{/if}
-			{if $paymentsEnabled && $acceptSubscriptionPayments && $subscriptionStatus == \APP\subscription\Subscription::SUBSCRIPTION_STATUS_AWAITING_ONLINE_PAYMENT}
+			{if $acceptSubscriptionPayments && $subscriptionStatus == \APP\subscription\Subscription::SUBSCRIPTION_STATUS_AWAITING_ONLINE_PAYMENT}
 				<p class="subscription_disabled">{translate key="subscriptions.status.awaitingOnlinePayment"}</p>
-			{elseif $paymentsEnabled && $acceptSubscriptionPayments && $subscriptionStatus == \APP\subscription\Subscription::SUBSCRIPTION_STATUS_AWAITING_MANUAL_PAYMENT}
+			{elseif $acceptSubscriptionPayments && $subscriptionStatus == \APP\subscription\Subscription::SUBSCRIPTION_STATUS_AWAITING_MANUAL_PAYMENT}
 				<p class="subscription_disabled">{translate key="subscriptions.status.awaitingManualPayment"}</p>
+			{elseif $subscriptionStatus != \APP\subscription\Subscription::SUBSCRIPTION_STATUS_ACTIVE}
+				<p class="subscription_disabled">{translate key="subscriptions.inactive"}</p>
 			{elseif $individualSubscription->isNonExpiring()}
 				<p class="subscription_active">{translate key="subscriptionTypes.nonExpiring"}</p>
 			{elseif $individualSubscription->isExpired()}
