--- a/lib/pkp/templates/frontend/pages/userRegister.tpl
+++ b/lib/pkp/templates/frontend/pages/userRegister.tpl
@@ -115,18 +115,28 @@
 		{include file="frontend/components/registrationFormContexts.tpl"}
 
 		{* When a user is registering for no specific journal, allow them to
-		   enter their reviewer interests *}
+		   enter their reviewer interests, if any journal offers a reviewer role *}
 		{if !$currentContext}
-			<div class="fields">
-				<div class="reviewer_nocontext_interests">
-					<label>
-						<span class="label">
-							{translate key="user.register.noContextReviewerInterests"}
-						</span>
-						<input type="text" name="interests" id="interests" value="{$interests|default:""|escape}">
-					</label>
+			{assign var=siteCanRegisterReviewer value=false}
+			{foreach from=$reviewerUserGroups item=contextReviewerUserGroups}
+				{foreach from=$contextReviewerUserGroups item=userGroup}
+					{if $userGroup->permitSelfRegistration}
+						{assign var=siteCanRegisterReviewer value=true}
+					{/if}
+				{/foreach}
+			{/foreach}
+			{if $siteCanRegisterReviewer}
+				<div class="fields">
+					<div class="reviewer_nocontext_interests">
+						<label>
+							<span class="label">
+								{translate key="user.register.noContextReviewerInterests"}
+							</span>
+							<input type="text" name="interests" id="interests" value="{$interests|default:""|escape}">
+						</label>
+					</div>
 				</div>
-			</div>
+			{/if}
 
 			{* Require the user to agree to the terms of the privacy policy *}
 			{if $siteWidePrivacyStatement}
